style.css

```diff
@@ -10,13 +10,15 @@ body {
 }
 .header {
   display: flex;
-  justify-content: space-between;  
+  justify-content: space-between;
+  margin-bottom: 100px;  
 }
 .logo {
   position: relative;
+  margin: 0 0 0 5px;
   padding: 40px 0;
   padding-left: 44px;
-  font: bold 24px/32px pt_serif, arial, helvetica, sans-serif;
+  font: 700 24px/32px PTSerif,arial,helvetica,sans-serif;
```

Comment:
Codacy found an issue: [Expected single space after "," in a single-line list (value-list-comma-space-after)](https://app.codacy.com/gh/WebHeroSchool/HTML_CSS_2.0-Artem1998/pullRequest?prid=6242157)